browser/unstoppable_domains/unstoppable_domains_service_factory.cc

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/unstoppable_domains/unstoppable_domains_service_factory.h"
+
+#include <memory>
+
+#include "brave/browser/brave_browser_process_impl.h"
+#include "brave/browser/unstoppable_domains/unstoppable_domains_service_delegate_impl.h"
+#include "brave/components/unstoppable_domains/unstoppable_domains_service.h"
+#include "brave/components/unstoppable_domains/utils.h"
+#include "components/keyed_service/content/browser_context_dependency_manager.h"
+
+namespace unstoppable_domains {
+
+UnstoppableDomainsServiceFactory::UnstoppableDomainsServiceFactory()
+    : BrowserContextKeyedServiceFactory(
+          "UnstoppableDomainsService",
+          BrowserContextDependencyManager::GetInstance()) {}
+
+UnstoppableDomainsServiceFactory::~UnstoppableDomainsServiceFactory() {}
+
+// static
+UnstoppableDomainsServiceFactory*
+UnstoppableDomainsServiceFactory::GetInstance() {
+  return base::Singleton<UnstoppableDomainsServiceFactory>::get();
+}
+
+// static
+UnstoppableDomainsService* UnstoppableDomainsServiceFactory::GetForContext(
```

Comment:
do we want it in incognito? By default the service will not be created

Reply:
@iefremov I don't think we need this service in incognito for what it currently does.